rt/src/runtime.rs

```diff
@@ -28,7 +28,7 @@ use std::io::{stdout, Write as _};
 use std::process::exit as rust_exit;
 use std::thread;
 #[cfg(unix)]
-fn ignore_sigpipe() {
+extern "C" fn ignore_sigpipe() {
```

Comment:
This change doesn't seem necessary. What I meant was to use `extern "C"` to define the `signal()` foreign function, similar to the example at https://doc.rust-lang.org/nomicon/ffi.html#calling-foreign-functions. For the constants you can just take the values from the libc crate, as they're the same across Unix platforms.

Reply:
ahh finally I got your point. Sorry for confusion :sweat_smile: